### ISSUE TITLE
Is there a way to repeat the video instead of exiting full screen?

### ISSUE THREAD

**Issue by Fr333m4n** [jepiqueau/capacitor-video-player#76]
Hello,

I'm trying to play a mp4 in full screen and to repeat it until I manually exit the view.
Is there a way to achieve that or it would need some change in the lib?

**Comment by jepiqueau** [comment]
@Fr333m4n yes since the last version you set the exitOnEnd parameter to false in the initPlayer options

**Comment by Fr333m4n** [comment]
I've tried, the player does not exit, yes. But It does not repeat either.
I have to hit the play button again.

**Comment by jepiqueau** [comment]
@Fr333m4n yes you have to start it again.  We could add an option to look even if i do not see the purpose. May be you can explain a bit

**Comment by Fr333m4n** [comment]
Sure, I search for short videos, less than 15 seconds, and usually, they are loops, intended to be played many times. They are a bit like GIFs, but with sound.
For instance, a meme video: https://media.vlipsy.com/vlips/Z1tUOMuh/480p.mp4

**Comment by jepiqueau** [comment]
@Fr333m4n this has been fixed in 3.4.0-4.
```
      const res: any  = await this.videoPlayer.initPlayer({mode: 'fullscreen',url: this.mUrl, subtitle: this.mSturl,
                                                          language: this.mStlang, subtitleOptions: this.mStoptions,
                                                          playerId: 'fullscreen', rate: this.mRate, exitOnEnd: false,
                                                          loopOnEnd: true, componentTag:'app-fullscreen'});

```

**Comment by Fr333m4n** [comment]
This is working great, thanks!
